otter/integration/tests/test_convergence.py

```diff
@@ -1210,3 +1297,21 @@ def _only_delete_clb_while_scaling(cls, name, method):
 copy_test_methods(
     ConvergenceTestsWith1CLB, ConvergenceTestsWith2CLBs,
     filter_and_change=ConvergenceTestsWith2CLBs._only_delete_clb_while_scaling)
+
+def statusMatches(s, capacities=None):
```

Comment:
This would only be applicable if the group is active and desired is 0, or the group is in error somehow (because the pending capacity has to be equal to the desired capacity, as the function is currently defined).  Can we make this more general and just check that the state matches a given status, pending, and desired?

(Also, the group can only be in ACTIVE or ERROR - BUILD status only applies to Nova servers).